### PR TITLE
PC LTP runtime python version default

### PR DESCRIPTION
**Pull request by m-dati**
New _python_ runner for public cloud LTP tests set as default _runltp-ng_ and removed the existing _perl_  version

- Related ticket: https://progress.opensuse.org/issues/116668
- Needles: none
- Verification run: TBD

### CODE REVIEW

File: tests/publiccloud/run_ltp.pm [hunk 1]
```diff
@@ -53,15 +53,9 @@ sub upload_ltp_logs
...
-        $log_file = Mojo::File::path('ulogs/result.json');
-        upload_logs("$root_dir/result.json", log_name => $log_file->basename, failok => 1);
-    } else {
-        # default ltp runtime: perl
-        $log_file = Mojo::File::path('ulogs/ltp_log.json');
-        upload_logs("$root_dir/ltp_log.raw", log_name => 'ltp_log.raw', failok => 1);
-        upload_logs("$root_dir/ltp_log.json", log_name => $log_file->basename, failok => 1);
-    }
+
+    $log_file = Mojo::File::path('ulogs/result.json');
```
Comment (asmorodskyi) [inline tests/publiccloud/run_ltp.pm:57]:
```suggestion
    my $log_file = Mojo::File::path('ulogs/result.json');
```

Comment (asmorodskyi) [reply tests/publiccloud/run_ltp.pm:57]: and you can delete separate declaration of this war two lines above

**Comment by m-dati** [comment]
The last commit contains in _tests/publiccloud/run_ltp.pm_ the default use of ` acerv/runltp-ng#master` for PC LTP tests.
I should be merged to the master of the os-autoinst-distri-opensuse/  after that the tests in [group 447](https://openqa.suse.de/group_overview/447) are ok for a while.

**Comment by m-dati** [comment]
Here are 4 VR tests in group [447](https://openqa.suse.de/tests/overview?distri=sle&groupid=447&build=%3A26832%3Agrub2&version=12-SP5), using the PR branch  **runltp-python** :

pc_ltp_py_syscalls@md1@gce_n1_standard_2 https://openqa.suse.de/t10050944

pc_ltp_py_cve@md2@gce_n1_standard_2  https://openqa.suse.de/t10050991

pc_ltp_py_syscalls@md3@az_Standard_A2_v2 https://openqa.suse.de/t10050992 

pc_ltp_py_cve@md4@az_Standard_A2_v2 https://openqa.suse.de/t10050995

**Pull request merged**